app/invocation/invocation_cancel_button.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Slash as SlashIcon } from "lucide-react";
+import { OutlinedButton } from "../components/button/button";
+import rpcService from "../service/rpc_service";
+import { invocation } from "../../proto/invocation_ts_proto";
+
+export interface InvocationCancelButtonComponentProps {
+  invocationId: string;
+}
+
+export default class InvocationCancelButtonComponent extends React.Component<InvocationCancelButtonComponentProps> {
+  private onClick() {
+    rpcService.service.cancelInvocation(
```

Comment:
nit: add an error handler in case the RPC fails (`.catch(e => errorService.handleError(e))`). This will show a red banner with the error message.

(We would like to make this error handling more automatic instead of having to do it manually for every RPC, but aren't quite there yet)

Reply:
Done